### ISSUE TITLE
Reverted solution no longer compiles

### ISSUE THREAD

**Issue by jol64** [dotnet/upgrade-assistant#1539]
### Describe the bug

After performing https://github.com/dotnet/upgrade-assistant/issues/1537 and https://github.com/dotnet/upgrade-assistant/issues/1538 I reverted all changes and restarted Visual Studio. However the solution does no longer compile.

Guys, reverting in a source code control system is supposed to undo all changes performed without leaving any state. Fortunately I do have backups.

### Further technical details
see referenced issues.

**Comment by SaileshBK** [comment]
what kind of error you are getting?

**Comment by joachimlindenberg** [comment]
iirc all .NET framework types were just unknown.

**Comment by SaileshBK** [comment]
Did you try manually removing bin and obj and unload and reload the project ?

**Comment by joachimlindenberg** [comment]
No. I tried solution clean. Manually removing bin and obj - 80 steps? Sorry, I lost two working days because of the shit.
